pkg/types/types.go

```diff
@@ -67,9 +67,10 @@ type NetConf struct {
 
 // Result is what gets returned from the plugin (via stdout) to the caller
 type Result struct {
-	IP4 *IPConfig `json:"ip4,omitempty"`
-	IP6 *IPConfig `json:"ip6,omitempty"`
-	DNS DNS       `json:"dns,omitempty"`
+	IP4 		*IPConfig `json:"ip4,omitempty"`
+	IP6 		*IPConfig `json:"ip6,omitempty"`
+	DNS 		DNS       `json:"dns,omitempty"`
+	CUSTOMATTR  	*CUSTOM    `json:"custom,omitempty"`
 }
```

Comment:
I agree with @dcbw's here and suggest to use `Args`, so there is consistency with the recently introduced change to the input configuration. Resulting in this line
```Go
  	Args  	map[string]interface{}    `json:"args,omitempty"` 
```

Although on top of my head I'm not sure if we need to value type to be of `interface{}` or if `string` would suffice.